src/components/header/Header.jsx

```diff
@@ -17,10 +18,14 @@ export const Header = () => {
   const navClasses = createClassNames(styles.nav, 'nav nav-fill');
   const ulClasses = createClassNames(styles['nav-tabs'], 'nav nav-tabs');
   const balanceClasses = createClassNames('nav-item', 'nav-link', styles.balance);
+  const burgerMenuClasses = createClassNames(styles['burger-menu']);
```

Comment:
also don't need creating redundant constants, use it in attributes
className={styles.burgerMenu}>
and don't wrap it with createClassName if there is only one element